src/NuGet.Indexing/NuGet.Indexing.csproj

```diff
@@ -197,7 +197,7 @@
       <Version>4.3.3</Version>
     </PackageReference>
     <PackageReference Include="WindowsAzure.Storage">
-      <Version>8.2.1</Version>
+      <Version>9.3.3</Version>
```

Comment:
NuGet.Services.Storage depends on WindowsAzure.Storage >= 9.3.3. This aligns with that version constraint.